tests/admin_custom_urls/urls.py

```diff
@@ -3,5 +3,5 @@
 from .models import site
 
 urlpatterns = [
-    url(r'^admin/', include(site.urls)),
+    url(r'^admin/', include(site, 'admin_custom_urls')),
```

Comment:
I tend to think specifying the namespace parameter name, e.g. `(site, namespace='admin_custom_urls')`, would be more readable. What do you think?